concent_api/core/management/commands/concent_adapt_data_to_current_version.py

```diff
@@ -0,0 +1,29 @@
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+from typing import Any
+
+from common.helpers import deserialize_message
+from core.models import StoredMessage
+from core.models import Subtask
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Adapts whole database to current Concent's version"
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        for subtask in Subtask.objects.all():
+            try:
+                report_computed_task =  deserialize_message(subtask.report_computed_task.data.tobytes())
+                if not isinstance(report_computed_task.size, int):
+                    delete_unsupported_messages(subtask.subtask_id)
+            except:
```

Comment:
It should be golem `MessageError`. Or do we expect other exceptions?

Reply:
Done